### PR TITLE
Recommended prerequisites now has its own page with editable fields

### PR DESCRIPTION
**Pull request by axelbjo**
- New admin page for editing recommended prerequisites for a course has been added. The behaviour is the same as for the 'Other information' section.
- Added some simple tests

### CODE REVIEW

**Review by karlandindrakryggen — APPROVED** [review]

**Review by amirhossein-haerian — APPROVED** [review]

File: i18n/messages.en.js [hunk 1]
```diff
@@ -192,10 +192,36 @@ module.exports = {
...
+    pageHeader: 'Recommended prerequisites',
+    step1: {
+      title: 'Edit text',
+      intro: `The heading “Recommended prerequisites” should describe what knowledge and skills (in addition to the eligibility requirements) the students need to be able to take the course. 
+              Students can use the information to prepare for the course or as a basis for choosing the course or not.
+              Preferably state explicit knowledge and skills and not just course names, for example “programming in Python” or “boundary value calculations”.`,
+      alert: `Note that the texts are displayed for all course offerings. If the course is offered at multiple times in the same semester, you may need to customize the text. Specify a maximum of 2000 characters per text.`,
+      nextButton: 'Preview',
+      fields: {
+        recommendedPrerequisitesSv: 'Recommended prerequisites (SW)',
```
Comment (allazis) [inline i18n/messages.en.js:210]: The `(SW)` should be `(SV)` if we're using standard language codes.

**Review by allazis — CHANGES_REQUESTED** [review]
Good job!

Only one small comment on the language code.

I've tested the publishing functionality and can't see the published text under `Recommended prerequisites` in the course info in REF. Is that expected behaviour at this point? :)

**Comment by axelbjo** [comment]
> Good job!
> 
> Only one small comment on the language code.
> 
> I've tested the publishing functionality and can't see the published text under `Recommended prerequisites` in the course info in REF. Is that expected behaviour at this point? :)

File: i18n/messages.en.js (hunk 1 above)
Comment (axelbjo) [reply i18n/messages.en.js:210]: That should be expected! The published text under recommended prereuisites is being fetched from Kopps in REF whereas it's being fetched from Om Kursen (kursinfo-api) in the new version (which has not been merged yet)

Comment (allazis) [reply i18n/messages.en.js:210]: Super! Only the language code then. 

Well done!

**Review by allazis — APPROVED** [review]

File: i18n/messages.en.js (hunk 1 above)
Comment (axelbjo) [reply i18n/messages.en.js:210]: > The `(SW)` should be `(SV)` if we're using standard language codes.

I'm wondering if it is supposed to be like that by design? It's something that's being displayed to the user and it changes to (SV) when you change the language on the page to Swedish

Comment (axelbjo) [reply i18n/messages.en.js:210]: > Super! Only the language code then.
> 
> Well done!

Thank you! :raised_hands:

Comment (allazis) [reply i18n/messages.en.js:210]: > > The `(SW)` should be `(SV)` if we're using standard language codes.
> 
> I'm wondering if it is supposed to be like that by design? It's something that's being displayed to the user and it changes to (SV) when you change the language on the page to Swedish

Interesting! Sounds like it's "planted" from before then. 

However, I would be careful using SW for Swedish as SW stands for Swahili according to the ISO 639 (you know language code widely used when choosing language in dropdowns and such) 😃 This doesn't effect the functionality so its a minor issue. But I would propose a correction of this, maybe as a new issue in Jira? Leave it for now and we can talk about it tomorrow at standup! Sorry about the fussiness over a some what not related matter!

Comment (axelbjo) [reply i18n/messages.en.js:210]: > > > The `(SW)` should be `(SV)` if we're using standard language codes.
> > 
> > 
> > I'm wondering if it is supposed to be like that by design? It's something that's being displayed to the user and it changes to (SV) when you change the language on the page to Swedish
> 
> Interesting! Sounds like it's "planted" from before then.
> 
> However, I would be careful using SW for Swedish as SW stands for Swahili according to the ISO 639 (you know language code widely used when choosing language in dropdowns and such) 😃 This doesn't effect the functionality so its a minor issue. But I would propose a correction of this, maybe as a new issue in Jira? Leave it for now and we can talk about it tomorrow at standup! Sorry about the fussiness over a some what not related matter!

That's a good point actually :smile: 
No worries at all, good that you brought it up! We can bring it up tomorrow.

**Pull request merged**